app/src/main/java/org/sopt/and/SignInActivity.kt

```diff
@@ -0,0 +1,318 @@
+
+        // 비밀번호 입력 필드
+        PasswordField(
+            value = passwordText,
+            onValueChange = { passwordText = it },
+            showPassword = showPassword,
+            onVisibilityChange = { showPassword = !showPassword }
+        )
+
+        Spacer(modifier = Modifier.height(30.dp))
+
+        // 로그인 버튼
+        SignInButton(text = "로그인", onClick = { onSignInAttempt(emailText, passwordText) })
+
+        Spacer(modifier = Modifier.height(30.dp))
+
+        // 아이디 찾기, 비번 재설정, 회원가입 버튼
+        ActionLinks(onSignUpClick = onSignUpClick)
+
+        Spacer(modifier = Modifier.height(55.dp))
+
+        Text(
+            text = "또는 다른 서비스 계정으로 가입",
+            color = Color.White,
+            modifier = Modifier.fillMaxWidth(),
+            textAlign = TextAlign.Center
+        )
+
+        // SNS 버튼들
+        Spacer(modifier = Modifier.height(20.dp))
+
+        SNSButtons()
+
+        Spacer(modifier = Modifier.height(15.dp))
+
+        // 주의사항 텍스트
+        Text(
+            text = "• SNS계정으로 간편하게 가입하여 서비스를 이용하실 수 있습니다. 기존 POOQ 계정 또는 Wavve 계정과는 연동되지 않으니 이용에 참고하세요.",
+            color = Color.Gray,
+        )
+    }
+}
+
+@Composable
+fun TopBar() {
+    Row(
+        modifier = Modifier
+            .fillMaxWidth(),
+        horizontalArrangement = Arrangement.SpaceBetween
+    ) {
+        Icon(
+            modifier = Modifier
+                .size(35.dp)
+                .clickable {  },
+            imageVector = Icons.AutoMirrored.Filled.KeyboardArrowLeft,
+            contentDescription = "뒤로가기",
+            tint = Color.White
+        )
+        Text(
+            modifier = Modifier
+                .weight(1f)
+                .align(Alignment.CenterVertically),
+            fontWeight = FontWeight.Bold,
+            text = "Wavve",
+            textAlign = TextAlign.Center,
+            color = Color.White,
+            fontSize = 30.sp,
+        )
+
+        Spacer(modifier = Modifier.size(35.dp))
+    }
+}
+
+@Composable
+fun InputField(value: MutableState<String>, placeholder: String, onValueChange: (String) -> Unit) {
+    TextField(
+        value = value.value,
+        onValueChange = { newText -> value.value = newText; onValueChange(newText) },
+        modifier = Modifier
+            .fillMaxWidth()
+            .height(60.dp),
+        placeholder = { Text(text = placeholder, color = Color.Gray) },
+        colors = TextFieldDefaults.colors(
+            focusedContainerColor = Color.DarkGray,
+            unfocusedContainerColor = Color.DarkGray,
+            focusedTextColor = Color.White,
+            unfocusedTextColor = Color.White,
+            focusedIndicatorColor = Color.Transparent,
+            unfocusedIndicatorColor = Color.Transparent
+        ),
+        shape = RoundedCornerShape(5.dp),
+        singleLine = true
+    )
+}
+
+@Composable
+fun PasswordField(
+    value: String,
+    onValueChange: (String) -> Unit,
+    showPassword: Boolean,
+    onVisibilityChange: () -> Unit
+) {
```

Comment:
비밀번호를 입력하는 텍스트필드의 역할을 하는 함수이니까 사용자가 show버튼을 누르기 전까지는 웬만하면 showPassword값이 false일거라고 생각합니다! 이럴 때는 기본값을 주면 더 편리하게 사용할 수도 있을 것 같아요!!
```suggestion
fun PasswordField(
    value: String,
    onValueChange: (String) -> Unit,
    showPassword: Boolean=false,
    onVisibilityChange: () -> Unit
) {
```